js/capture.js

```diff
@@ -1,5 +1,9 @@
-/* eslint no-console: "off" */
-/* eslint no-unused-vars: "off" */
+/*
+* Listens for HTTP request responses, sending first- and
+* third-party requests to storage.
+*/
+
+capture.init();
```

Comment:
Despite the fact JS has hoisting, I prefer this after it's definition. However I'm gonna merge this.